data/cloudcraft.json

```diff
@@ -85,6 +85,7 @@
                   "type": "object",
                   "properties": {
                     "blueprints": {
+                      "description": "An array of blueprints without diagram data.",
```

Comment:
A bit above this, remove the "The response is an array of blueprint entries without the diagram data" sentence. Change this to "An array of blueprint metadata".